### PR TITLE
Fix style

### PR DESCRIPTION
**Pull request by tezc**
Fix style of functions in raft_log.c raft_node.c and raft_server_properties.c and their declarations in header files. 
Mostly pointer alignment and braces after if checks.

### CODE REVIEW

File: src/raft_server_properties.c [hunk 1]
```diff
@@ -127,81 +131,77 @@ const char *raft_get_state_str(raft_server_t* me)
...
-            raft_term_t term = ety->term;
-            raft_entry_release(ety);
-            return term;
-        } else if (me->snapshot_last_idx == current_idx) {
-            return me->snapshot_last_term;
-        }
+
+    if (current_idx == 0) {
+        return 0;
+    } else if (current_idx == me->snapshot_last_idx) {
```
Comment (fadidahanna) [inline src/raft_server_properties.c:196]:
```suggestion
    }
    
    if (current_idx == me->snapshot_last_idx) {
```

File: include/raft.h [hunk 2]
```diff
@@ -1276,27 +1276,27 @@ raft_node_id_t raft_node_get_id(raft_node_t* me);
...
 /** Get the most recent log's term
  * @return the last log term */
-raft_term_t raft_get_last_log_term(raft_server_t* me);
+raft_term_t raft_get_last_log_term(raft_server_t *me);
 
 /** Turn a node into a voting node.
  * Voting nodes can take part in elections and in-regards to committing entries,
  * are counted in majorities. */
-void raft_node_set_voting(raft_node_t* node, int voting);
+void raft_node_set_voting(raft_node_t *me, int voting);
```
[the remark this answers is not in the document]
Comment (tezc) [reply include/raft.h:1291]: > To be honest, I'm so happy with the me notation,

Are you "so happy" or "not happy"? :) 
I think we can change it to `node`. As we are going to destroy blame log anyway.

[the remark this answers is not in the document]
Comment (fadidahanna) [reply include/raft.h:1291]: yes, sorry - typo, of course it's not so happy :(, anyway - feel free to choose what you think it's better, but let's be consistent.

[the remark this answers is not in the document]
Comment (tezc) [reply include/raft.h]: done.

**Review by fadidahanna — APPROVED** [review]

**Pull request merged**